### DISCUSSION TITLE
Deliverable 1 : Review Report

### DISCUSSION THREAD

**Discussion by mankoff** [IACS-cryo/Delineation-WG#20] [General]
Let's get started... our first deliverable is a review report. 

> Review report outlining the current situation, extracting the relevant information and
suspected double counting in previous studies and IPCC report and proposed solutions
to improve the situation.

I've created a Milestone for this, and we can add specific action items there as they filter out of this discussion: https://github.com/IACS-cryo/Delineation-WG/milestone/3 

+ [X] The `current situation` is outlined in our proposal
+ [ ] `Suspected double counting` can be defined more clearly, perhaps with order-of-magnitude estimates?
+ [X] `proposed solutions` is likely already mostly defined in the proposal.

One way to complete this deliverable (D1) could be to just start the work, and have the `suspected double-counting` be a by-product of later deliverables. 

As for `just start the work`, what does that actually mean? I'm not sure, but suggest a two-pronged approach, top-down and bottom up. We can start small (@fmaussion suggested Thurston Island) and also big (Geike Plateau? Antarctica or Greenland in general?) and see what develops from these two approaches.

Of [our 40 Team Members](https://docs.google.com/spreadsheets/d/18DBHRbEAuquOqChHN2GqUR2uPKpW7D1Y-IBqMPM6TtM/edit#gid=1372946030) 20 people (not necessarily a subset) are monitoring this GitHub repository and should get this email. You can reply to this email, reply with some more fidelity on GitHub, or unsubscribe to this specific email if this topic is not relevant to you.

**Comment by willkochtitzky** [comment]
Thanks Ken! Would it be useful to have a zoom call to start this work? I am happy to help, just don't know where to begin.

**Reply by bearecinos** [reply]
I agree I would like to get involve but I don't have a clear vision for  `Suspected double counting point` for example and/or how would one begin to assess that, further discussion on that matter or in general about how to get involve would be helpful.

**Reply by mankoff** [reply]
If possible I would prefer to decide what/how here rather than Zoom. We're spread across time zones (N. America, Europe, and India (NOTE: Should we open a new issue how to get broader community - Asia, Africa, & S. Hemisphere are not or under represented? They all work in GL & AQ too...)).

If we meet on Zoom someone will either be left out or awake in the middle of the night, we'll have to schedule it, the summary might make it back here for the many who don't attend, but the process will be transient and likely not well captured.

Let's do Zoom if an asynchronous text-based discussion fails, but let's try not to let that happen.

I would address `suspected double counting` by reading [Hock (2023)](http://doi.org/10.1017/jog.2023.1) and thinking about their methods but applying it to various ice sheet masks rather than global ice inventories. I think this boils down to a few spatial set queries on a few data sets.

What about a graphic like this:

![image](https://github.com/IACS-cryo/Delineation-WG/assets/145117/e354c939-4b18-4453-bbfa-499c6e5fd47b)

That's a bad rough draft. If those are the numbers then axis labels should be "inside" not "outside", where A is 100 % inside A, A might be 95 % inside B, and B might be 97 % inside A. Probably more useful to keep the axes labels as they are, and have the numbers be small. The diagonal would be 0, and we would report "3 % of A is outside B" or "180 km^2 of B is outside A".

If that seems like a useful summary, we need to decide which products we want to include, then do some spatial queries.

I'd also make it more detailed - MAR could have 3 nested sub-categories(columns and rows) of "33 %,  50 %, 66 %"  (it's a floating point ice mask) and BedMachine and others with sub-categories of CL# or "ice cap" or whatever classifications that product provides. RGI of course needs sub-categories too.

Numbers can be % or sq. km. Do it for AQ & GL. Can't easily do it by AQ or GL sector/region/etc. without picking one as the baseline which may be problematic.

Once we've quantified and defined area of every product overlap, underlap, inside, and outside with respect to every other product, are we done with this step?

Someone could then go into existing papers/products and see what the mass loss from those areas are for various products, which tells us under- or double-counting of mass (and potentially SLR), but I'm not sure this is a necessary step for this project. It is also likely not possible - that level of detail and reproducibility and openness is not always provided, and even when it is, the work for 3rd parties to get the answer may be prohibitive. I'd suggest we move forward rather than look back, and discuss how to start agreeing on a classification where the products disagree, (possibly re-classifying even if everything does agree, if they agree for the wrong reason), and how discuss how to start making adjustments so that we can do proper intercomparsions.

At project conclusion is the goal to re-generate this graphic with updated products (MAR vX, BedMachine vY, etc.) and have everything be 0?

Anyway, this is just one idea I thought of in response to the question above. Feel free to head to this issue and vote (thumbs up or down) if you think this is a valuable next step or not, or suggest other useful starting activities.

**Reply by mstudinger** [reply]
I'd like to second Ken's excellent point that having the discussion on this channel is more inclusive and transparent than a Zoom meeting. I was at the NASA TOPS Symposium all week (full disclosure: I get paid to promote TOPS: https://nasa.github.io/Transform-to-Open-Science/). We discussed this very issue at the meeting and there was general consensus that an open discussion on GitHub is the preferred way, but synchronous discussions can of course be done if needed. **The Delineation-WG, BTW, is a shining example for open science!**

**Comment by KristianKjeldsen** [comment]
I’ll second that, I’m happy to get involve and I agree an online meeting would be a good starting point to spark the process.
Best,
Kristian

**Reply by regine** [reply]
Regarding zoom: For other IACS working groups we had occassional zoom meeting (at least once a year) and typically offered 2 time slots so somewhat convenient time slots were available to anybody in the world. We also tried to meet (though always a subset) at major conference (IGS, AGU, EGU, IUGG). These in person and zoom meetings have been very useful to get folks engaged and move things forward.

I am also a bit confused what the next steps are and what you suggest. I see a need of a "stocktake" of the current issue so that the different communities understand how/why which masks are used; essentially a far more detailed description of the rationale of the different choices and implementations than in Hock et al., 2023. Is this what you have in mind ?

**Reply by mankoff** [reply]
I get the impression you have a specific vision of next steps. Can you clarify what you think should happen? What does `stocktake of the current issue` mean to you?

**Comment by JonathanCarrivick** [comment]
I am also happy to help out with the spatial analysis where and when I can.... but before that I think that it would it make sense to develop a clear workflow/protocol first, which probably requires working backwards from what output we want, and then all those able to help can get going on desiganted regions/datasets...

**Comment by hgoelzer** [comment]
The idea to have two or three zoom calls just to kick off the activity, match some faces to names and get a bit of an idea of people's background, what they are up to and what they bring to the table seems very useful to me. I am fully supporting that the bulk of the discussion and exchange then happens here on github. Having a zoom call once in a while and talking on conferences and workshops with people that happen to be there anyway appears to me like an almost necessary ingredient to build trust and desire to work together. But maybe I am getting old.

**Reply by mstudinger** [reply]
Absolutely! Needless to say I very much enjoyed the in-person conversations with folks at the TOPS Symposium. There is no substitute for that. Recording virtual/hybrid meetings and making the recordings available helps building that community. As an aside and not relevant to this group, we also discussed the pros and cons of people having to sign up for GitHub in order to participate in open science. Some folks don't like GitHub and we need to make sure they are included. Needless to say, we didn't come up with a solution...  and as you say, sometimes to get things started on a tight schedule, meetings are the best way to kick things off. NASA TOPS got "launched" that way but now we felt it's time to transition into a more inclusive and transparent (but slower) mode of community interaction.

**Reply by regine** [reply]
I don´t think zoom meetings are a problem in terms of inclusiveness (esp if 2 sessions run 12 hours apart are offered and the sessions are recorded and made available, as we have done in previous WGs). Sorting things out face to face (esp settling misunderstandings, brainstorming ideas etc) is undeniably more efficient than written back and forth.

On the other hand, I´d be careful with the assumption that using "github" is an all-inclusive-way. In fact I would argue it is problem and also for this WG. Some have not even signed up yet (so they are currently excluded from the discussions), and it is definitely a barrier for those not familiar and comfortable with it and as such discrimatory, and it may make it also harder for adding more members in the future. Saying they can just sign up and learn it is not the answer. I am not saying we should abondon github but perhaps one should address this issue somehow and make it easier for those who experience this as a significant barrier.

**Reply by mankoff** [reply]
@regine we can do Zoom. I'll send out a poll. 

I agree GitHub is a barrier - but barriers exist everywhere. We're not removing all, just trying to minimize them. Perhaps we can do an intro to GitHub at some point, but many tutorials exist already.

But prior to zooming, I'd like you to write, here, what you are expecting and looking for. You have a lot of experience in this area, and some opinions. Specific suggestions would be much appreciated.

If not, we'll likely end up doing something slightly different, based on initial comments above.

**Reply by fmaussion** [reply]
It's fine to have a zoom, but I suspect it won't be the solution to sort things out, and I'd rather have people exchanging here rather than waiting for the zoom call to occur. We have a difficult task ahead, and I don't think anyone here has yet a clear vision of how "this needs to be done".  Messages here will help to prepare and brainstorm ahead of the meeting.

**Reply by regine** [reply]
Ken, here what I mean by "stocktake": essentially not much different than Deliverable D1 in the proposal "Review report outlining the current situation, extracting the relevant information and suspected double counting in previous studies and IPCC report and proposed solutions to improve the situation." My impression is there is not much across-discipline awareness of what masks are used for the difference communities (ice sheet, glacier, SMB models, full evolution models ...). So as a first step it is probably useful to summarize what all the studies have done and why. For the Hock et al., 2023 we tried to figure how what folks did and it was very difficult, information missing entirely or hidden deeply in supplementary or even when contacting authors they did not know. Extracting all that information how things have been done and the rationale for the choices of the masks and summarizing it could be a paper and is needed before any changes to masks can be done. Essentially which does did take which mask and why ? This will then allow identifying overlap and double-counting. In practice WG members from these different communities would work towards summarizing their part (glacier/ice sheet assessments (past), glacier projections, climate models (SMB, past/future), ice sheet projections.

**Reply by fmaussion** [reply]
I agree with @regine that "across-discipline awareness" is perhaps one of the best potential outcome of D1. I fear however that a precise counting, literature and global stocktake will be tedious and that the task will be difficult to federate. This sounds like a very slow process and, as always, we are unpaid. I would be quite keen if there were a clear "task leader" for this, for example a post-doc paid to do exactly this for a few months...

**Reply by mankoff** [reply]
@regine I disagree with the statement:

> Extracting all that information how things have been done and the rationale for the choices of the masks and summarizing it [...] is needed before any changes to masks can be done. 

I think this would be useful to do, but given the complexity and challenges you had doing this in Hock _et al._ (2013), I question if it is needed as you state above, and I don't yet see why it is necessary.

Specifically, I don't think it is necessary (or likely possible within reasonable work efforts) to quantify the mass double counting for multiple historical publications. We can probably get the old masks and quantify area of overlap. In the future, a study can use old mask (with overlap) and new mask generated by this WG w/o overlap, and quantify the mass of double-counting given their (future) inputs. I'm also not sure if/why this needs to be a comprehensive literature review as you did, or just one example per mask.

In general, I would prefer to minimize back-work (discussing existing products), although there must be some of this, and I prefer to focus on forward-facing work (adjusting existing products to avoid overlap seems easiest, then generating new products as needed).

**Reply by regine** [reply]
Maybe it´s semantics and we misunderstand each other. We can not look forward and come up with new masks and ways of avoiding double-counting if the different communities do not understand each other and what was/is done and why. The difference to Hock et al is that now we have a WG with members of all relevant communities. What I had in mind is not a super time/work-expensive exercise but that each "group" summarizes in a page or so which masks were taken, i.e. what spatial domain their estimates refers to and why, and if they did, how double-counting was considered/avoided. For each "group" a lead could be identified who then together with their colleagues writes this up for each topic. Each group should know what they did, so this should be easy. I did not have in mind reconstructing or quantifying double-counting for past studies. So the idea is an extension of Hock et al, but now harvesting all the knowledge from those different communities.
Without a basic understanding of what has been done in the past we can´t look forward and even after digging into this for the Hock et al paper I felt that I have little grasp on what people did (not all papers provide sufficient detail and/or hidden deeply in Supp Material).
The groups I see are: glacier assessments (past/recent changes), glacier projections, ice sheet mass change assessments, i.e. past/recent (e.g. GRACE, altimetry ...), ice sheet climatic mass balance past/future, ice sheet future evolution. Ice sheet groups separate for Greenland and Antarctica.
All these groups do their own things with somewhat little awareness of definition and rationale of choice of spatial domains of other groups. So, step one would be to raise this awareness. We can´t make things better if we lack an understanding of what was done before, how and why. Having this in one place (e.g. a resulting paper) would be super useful not only for us going forward but also for the different communities in their efforts.

**Comment by hgoelzer** [comment]
**Double counting – ISMIP6 and GlacierMIP2**

An attempt to visualise the problem of double counting from a practical point of view, and how we addressed it in ISMIP6/GlacierMIP for Greenland. With this, I mainly want to point out that even after being in possession of a land surface delineation, there are problems that arise from the difference between glacier and ice sheet models and their resolution contrast. 

We wrote (Goelzer et al., 2020):
“In agreement with the GlacierMIP exercise, we have attempted to remove the contribution of loosely connected glaciers and ice caps in the periphery of Greenland from our mass change estimates to avoid double-counting in global sea-level change assessments. This has been done by correcting the ice sheet mass change per grid cell by the area fraction of the glaciers (level 0–1) in the Randolph Glacier Inventory (RGI Consortium, 2017).” 

Figure 1 illustrates the masking approach described above. The implemented solution means that GlacierMIP2 excluded glaciers with connect=2 altogether. Mass changes from A and B were only reported from ISMIP6. The area fraction of connect=[0,1] glaciers in C and D is 2/16 and 6/16, respectively. While GlacierMIP2 reported full mass changes from these glaciers, ISMIP6 reported mass changes from C and D multiplied by 1-(2/16) and 1-(6/16), respectively.

<img width="600" alt="image" src="https://github.com/IACS-cryo/Delineation-WG/assets/35115552/de16ab6c-ac9d-4042-b8d3-ac803a979618">

Figure 1. Illustration of possible double counting between ice sheet and glacier models. The colours indicate classification of the land surface in RGI-60 (in reality these are non-gridded shape file contours). The frames A-D show the grid cells of an ice sheet model, all of which are ice covered in this example and are represented by one value of ice mass each. Glaciers have different levels of connectivity. 

I still have all the ice sheet model data and area fractions from ISMIP6 if this would be useful for quantifying the double counting. However, I am not sure we can do that without knowing what the correct answer should be. Maybe somebody here has some ideas for that?


**RGI60** (https://www.glims.org/RGI/00_rgi60_TechnicalNote.pdf)

Connect
The Connect attribute records the connectivity level developed by Rastner et al. (2012) for glaciers in Greenland. Glaciers that are physically detached from the ice sheet have a connectivity level of 0. A glacier is weakly connected if it is in contact with the ice sheet only at a well-defined divide in the accumulation zone, and strongly connected if the divide is indistinct in the accumulation zone and/or confluent with an ice-sheet outlet in the ablation zone:

Value Connect
0 No connection
1 Weak connection
2 Strong connection

Goelzer, H. at al 2020: The future sea-level contribution of the Greenland ice sheet: a multi-model ensemble study of ISMIP6, Cryosphere, 14, 3071–3096, https://doi.org/10.5194/tc-14-3071-2020.

**Comment by fmaussion** [comment]
Dear all, to add a bit more context: while the end goal outlined in the proposal is quite clear, the steps towards this goal are not (at least not to me). Deliverable 1 was suggested during proposal writing as a way to have a better grasp of the problem and the work to be done (mapping, agreeing, etc.), before being able to define the "clear workflow" that we all want (see also https://github.com/IACS-cryo/Delineation-WG/discussions/14).

Perhaps Deliverable 1 is not needed? Ken and I would welcome more opinions about this here.

Eventually, we will have to decide things such as: "is Thurston Island part of the RGI or not, and why?". I was thinking perhaps that some literature review and community input in a living document (the deliverable 1 report) might help to find out how to take this decision, although again, I'm quite open to hear about other ways to reach this decision.

**Comment by hgoelzer** [comment]
If the report includes information on what has been done in the past, here is more information from ISMIP6 Greenland. 

We have operated with 4 different delineation products that fullfill various purposes. 

-  Mass change estimates were reported (and used by AR6) on two basin sets that were derived from both IMBIE delineations (http://imbie.org/imbie-3/drainage-basins/; Rignot and Zwally). Fig. 1c shows only Rignot. Important modification and highly relevant for our discussion is that we had to extend the basins outward to account for ice sheet models that have a larger than observed footprint. 

To produce the forcing (Slater et al., 2020), we operated with two more delineations (Fig 1.a/b).

- To produce runoff estimates for our retreat forcing we used hydrological drainage basins  that were calculated based on BedMachine v3.

- To produce ocean thermal forcing, we used a large-scale delineation to distinguish ocean circulation regimes around Greenland. This was based on IMBIE Rignot basins, but modified for this purpose and extending to the ocean outside of Greenland. 

![image](https://github.com/IACS-cryo/Delineation-WG/assets/35115552/1299d068-4cc5-4fa7-af97-440ac86af6d7)
Figure 1. Basin delineation products used for ISMIP6 Greenland. a) hydrological drainage basins for runoff routing. b) ocean basins for thermal forcing. c) Extended IMBIE basins (Rignot) to report ice sheet mass changes. 

Slater, D. A., Felikson, D., Straneo, F., Goelzer, H., Little, C. M., Morlighem, M., Fettweis, X., and Nowicki, S.: Twenty-first century ocean forcing of the Greenland ice sheet for modelling of sea level contribution, Cryosphere, 14, 985-1008, https://doi.org/10.5194/tc-14-985-2020, 2020.
